sdk/identity/azure-identity/azure/identity/_credentials/imds.py

```diff
@@ -21,8 +21,8 @@
 
 PIPELINE_SETTINGS = {
     "connection_timeout": 2,
-    "retry_backoff_factor": 2,
-    "retry_backoff_max": 60,
```

Comment:
Why we delete backoff_max?

Reply:
Seemed like unnecessary configuration after the change to the retry_backoff_factor. A user would hit the retry limit well before this is hit, so I think it's fine to go with the pipeline default unless a user wants to configure it themselves.

Reply:
User can still customize retry_backoff_factor & retry_count. So it may still take effect. :)

Reply:
Then I'll just leave it as-is to minimize change.